### PR TITLE
Fix doc index page

### PR DESCRIPTION
**Pull request by dipinknair**
- [x] Update the animation as gif
- [x] Fix documentation index page
- [x] modify folder structure
- [x] Add Image to technology showcase example
- [x]  delete static images

### CODE REVIEW

**Review by koubaa — APPROVED** [review]

**Comment by RobPasMue** [comment]
Looks like there is an issue on the export https://github.com/ansys/pymechanical-embedding-examples/actions/runs/6223242209/job/16888808864?pr=78#step:8:2871

File: doc/source/index.rst [hunk 1]
```diff
@@ -5,9 +5,17 @@ Embedding examples for `PyMechanical <_pymechanical_docs>`_ are contained in thi
 
 .. === EXAMPLES Gallery ===
 
+..
+   We have to include this rather than include it in a tree.
+
+.. include:: examples/index.rst
+   :start-line: 2
+
```
Comment (Revathyvenugopal162) [inline doc/source/index.rst:13]: Including this is will create a duplication of pages for basic/tech examples and an example section, is this expected?
image

File: .gitignore [hunk 2]
```diff
@@ -166,4 +166,5 @@ doc/source/technology_showcase_examples/*.pickle
 
 # Results generated during doc build and example run locally
 examples/technology_showcase/out
-examples/basic/out
\ No newline at end of file
+examples/basic/out
```
Comment (Revathyvenugopal162) [inline .gitignore:169]: clean gitignore #L162 - #L165

File: doc/source/index.rst (hunk 1 above)
Comment (Revathyvenugopal162) [reply doc/source/index.rst:13]: i might also change the repetition of "examples" in the heading , basic and technology show cases. as in pymapdl-examples https://examples.mapdl.docs.pyansys.com/index.html

**Review by koubaa — APPROVED** [review]

File: doc/source/index.rst (hunk 1 above)
Comment (dipinknair) [reply doc/source/index.rst]: Fixed

**Pull request merged**